TTS/tts/models/vits.py

```diff
@@ -1127,12 +1127,12 @@ def inference_voice_conversion(
             self.config.audio.hop_length,
             self.config.audio.win_length,
             center=False,
-        ).transpose(1, 2)
+        )
         y_lengths = torch.tensor([y.size(-1)]).to(y.device)
         speaker_cond_src = reference_speaker_id if reference_speaker_id is not None else reference_d_vector
         speaker_cond_tgt = speaker_id if speaker_id is not None else d_vector
         # print(y.shape, y_lengths.shape)
-        wav, _, _ = self.voice_conversion(y, y_lengths, speaker_cond_src, speaker_cond_tgt)
+        wav, _, _ = self.voice_conversion(y.transpose(1, 2), y_lengths, speaker_cond_src, speaker_cond_tgt)
```

Comment:
Just remove the transpose(1,2) from lines 1135 and 1160.
I believe it is redundant.